registries/_format/sf-binary.md

```diff
@@ -0,0 +1,37 @@
+---
+owner: mikekistler
+issue: 
+description: structured fields byte sequence as defined in [RFC 8941]
```

Comment:
Is there a convention for references to RFCs? I'm using "RFC 8941" with a space between "RFC" and the number, but I see many places that omit that space, e.g. "RFC8941". Should I be using this instead?

Reply:
I slightly prefer the space, but I don't have strong feelings.  It is clear either way.

Reply:
According to the [RFC Style Guide](https://www.ietf.org/archive/id/draft-flanagan-7322bis-07.html#name-citations) it's with a space in plain text, and without space in a citation label within square brackets:

> Example: "See RFC 2119 [[BCP14](https://www.ietf.org/archive/id/draft-flanagan-7322bis-07.html#BCP14)] for more information."